nss-day-cohort-48/rare-client-polite-pirates
# All categories showing on DOM

#### Changes Made

1. Modified file `NavBar.js` to include "Category Manager" (with routing working (ApplicationViews))
2. added `Category.css`, `Category.js`, `CategoryList.js`, and `CategoryProvider.js`
   ​

#### Steps to Review

1. Checkout this branch locally.
   ```
   git fetch --all
   git checkout pk-categories
   ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
   > You will need to fetch and checkout this branch AS WELL AS my branch in my other PR (server-repo) to properly test 
      [link to other PR](https://github.com/nss-day-cohort-48/rare-server-polite-pirates/pull/12)
   > Confirm that the categories are listing in alphabetical order.
